frontend/src/components/ProductInfo.js

```diff
@@ -1,13 +1,13 @@
 import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
 
-export default function ProductInfo ({name, water, apiState}) {
-    ProductInfo.propTypes = {
-        name: PropTypes.string,
-        water: PropTypes.number,
-        apiState: PropTypes.string
-    }
+ProductInfo.propTypes = {
+    name: PropTypes.string,
+    water: PropTypes.number,
+    apiState: PropTypes.string
```

Comment:
nit: This is the state of a request not the api

Reply:
good point, thx!